Lucid/Core/CoreManager.swift

```diff
@@ -487,6 +622,120 @@ private extension CoreManager {
         }
     }
 
+    func get(withQuery query: Query<E>,
```

Comment:
As a follow up task (you might already have planned or started this)...

 We should either fully deprecate/delete the Combine functions or have them be simply wrapped so they actually call into the async/await functions so we don't have two sets of implementations.